doc/indices.sgm

```diff
@@ -68,6 +68,11 @@
             </para>
           </listitem>
         </itemizedlist>
+        <para>
+          GiST index can be also used for quickly finding the points closest to the given one
```

Comment:
Change `GiST index` to `A GiST index`. Sorry should have caught that in my previous review.